frontend/package.json

```diff
@@ -6,19 +6,20 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "http:server": "http-server dist -c-1",
-    "lint": "vue-cli-service lint --no-fix",
-    "lint:fix": "vue-cli-service lint"
+    "stylelint": "stylelint src/**/*.vue src/**/*.scss --syntax scss --cache",
+    "lint:fix": "vue-cli-service lint",
+    "lint": "vue-cli-service lint --no-fix && npm run stylelint"
   },
   "dependencies": {
     "@7ninjas/scss-mixins": "^1.0.1",
     "bootstrap": "4.1.3",
     "bootstrap-vue": "2.0.0-rc.11",
     "jwt-decode": "^2.2.0",
     "register-service-worker": "^1.5.2",
+    "uuid": "3.3.2",
     "vue": "2.5.17",
     "vue-router": "3.0.1",
-    "vuex": "^3.0.1",
-    "uuid": "3.3.2"
+    "vuex": "^3.0.1"
```

Comment:
We've discussed with @magul we shouldn't use "^", so maybe we could remove it here? :)

Reply:
`package-lock.json` takes care of that, although if this is a requirement, I am happy to comply. This PR doesn't change the version. NPM sorts imports alphabetically by default, so seems that it happened here too. 
If we are required to to use specific versions, lets make a separate pr that facilitates just that.